src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue

```diff
@@ -57,18 +57,22 @@
 			</div>
 		</div>
 		<div v-if="callIcon"
+			v-tooltip.auto="callIconTooltip"
 			class="participant-row__callstate-icon">
 			<Microphone
 				v-if="callIcon === 'audio'"
 				:size="24"
+				title=""
 				decorative />
 			<Phone
 				v-if="callIcon === 'phone'"
 				:size="24"
+				title=""
 				decorative />
 			<Video
 				v-if="callIcon === 'video'"
 				:size="24"
+				title=""
```

Comment:
🧐

Reply:
yep... otherwise there's a "title" tip that says "Video icon"... seems to be built-in

Reply:
Should not happen with the `decorative` attribute

Reply:
then probably a bug, when I remove the title attribute it looks like this:
<img width="178" alt="image" src="https://user-images.githubusercontent.com/277525/99512511-d6723780-2989-11eb-9773-135b1de51b34.png">

Reply:
ok, I managed to make it appear elsewhere as well:
<img width="197" alt="image" src="https://user-images.githubusercontent.com/277525/99513415-e9393c00-298a-11eb-8d9d-ce0d3d38da86.png">

According to the docs, the purpose of "decorative" is not to remove the title: https://github.com/robcresswell/vue-material-design-icons#props

unless it did before and it's a regression from a recent update

Reply:
nope, this also happens on Talk 10.0, so it was always there but no one hovered there long enough to see it.

I'll keep title empty here and make another PR to fix the other buttons.

Reply:
pushed https://github.com/nextcloud/spreed/pull/4633 (I also adjusted the titles there, so whichever is merged first wins)